tests/data/test_dataloader.py

```diff
@@ -686,7 +686,6 @@ def test_finetuning_dataloader_streaming(
         'dataset': {
             'max_seq_len': 2048,
             'decoder_only_format': True,
-            'allow_pad_trimming': False,
```

Comment:
why did you need to remove this arg? this is a valid dataset config key

Reply:
It was complaining this param, but not any more.